src/fixate/core/exceptions.py

```diff
@@ -1,32 +1,32 @@
-class SequenceAbort(BaseException):
+class SequenceAbort(Exception):
     pass
 
 
-class ScriptError(BaseException):
+class ScriptError(Exception):
     pass
 
 
-class TestRetryExceeded(BaseException):
+class TestRetryExceeded(Exception):
```

Comment:
Perhaps it grows the scope of this PR too much, but generally python exceptions will be suffixed with `Error`. We could rename them, keeping an alias for compatibility (but probably not worth the churn)